### DISCUSSION TITLE
we need steve

### DISCUSSION THREAD

**Discussion by steve-is-me** [FunkinCrew/Funkin#1650] [Ideas]
## What is your suggestion, and why should it be implemented?
![image](https://user-images.githubusercontent.com/83514968/116793833-143ec480-aac1-11eb-92e5-2abd22adf797.png)

**Comment by ghost** [comment]
No. This is not Minecraft. Please, stop

**Comment by aflacc** [comment]
s

**Comment by ghost** [comment]
[Steve in Friday Night Funkin](https://youtu.be/BFL9hlNZx1o)

**Comment by isakube** [comment]
> #### Please check for duplicates or similar issues before creating this issue.
> ## What is your suggestion, and why should it be implemented?
please stop doing this unless you have proof of you donating enough to receive the "you in-game" reward

this is funny but stop

**Comment by GraysonMakesGames** [comment]
funny but nope
